website/versions.json

```diff
@@ -1,4 +1,5 @@
 [
+  "v12.0.2",
```

Comment:
I don't think we want the website version updated here... 🤔. Double-check me but I think we only do this for forward versions:

https://github.com/nasa/cumulus/blob/master/docs/development/release.md#8-cut-new-version-of-cumulus-documentation

Reply:
https://github.com/nasa/cumulus/pull/3076/commits/cf019ba3f573988756c9eb373df7090bf6480c68

Reply:
Thanks!